<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MAP_MainWindow</class>
 <widget class="QWidget" name="MAP_MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MAP</string>
  </property>
  <widget class="QPushButton" name="pushButton_load_3D">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Load 3D Mesh ...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_mat_assemb">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Dataset Construction</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_load_smap">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Load sMAP data...</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_problem">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Poisson equation</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Helmholtz equation</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Heat equation</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Problem type :</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>97</x>
     <y>210</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Problem definition</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Solver definition</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_dir">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>101</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Direct solver</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_it">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>230</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Iterative solver</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_solve">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Solve</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_save_output">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>290</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save output...</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_solver">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>260</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Conjugate Grad.</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>OrthoDir</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>GMRES</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MinRes</string>
    </property>
   </item>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>3</width>
     <height>61</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_load_RHS">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Load RHS Data ...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_load_BC">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Load BC Data ...</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_homogeneous">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>141</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Homogeneous Problem</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_load_Mat">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>340</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Linear System</string>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>320</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_solver_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Conjugate Grad.</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>OrthoDir</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>GMRES</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MinRes</string>
    </property>
   </item>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>checkBox_dir</sender>
   <signal>clicked()</signal>
   <receiver>checkBox_it</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>163</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_it</sender>
   <signal>clicked()</signal>
   <receiver>checkBox_dir</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>60</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
